public-images/public-images-how-to/use-local-cloud-init-ds.rst

```diff
@@ -0,0 +1,96 @@
+.. _use-local-ds:
+
+Create and Use a Local Cloud-Init Datasource
+============================================
+
+Purpose
+-------
+
+The steps in this How-To guide will allow you to create a seed image that defines a custom cloud-init datasource.
+
+Using a custom local datasource is a great way to inject custom user data or metadata into cloud images launched
+locally, e.g. with `qemu`.
+
```

Comment:
Can we add one more line here about why one might want to do this, i.e. launch such customised images locally? Is it for some sort of testing?